public/app/features/admin/Users/UsersTable.tsx

```diff
@@ -0,0 +1,177 @@
+import { css } from '@emotion/css';
+import React, { useMemo } from 'react';
+import { DefaultSortTypes } from 'react-table';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  InteractiveTable,
+  CellProps,
+  Tooltip,
+  Icon,
+  useStyles2,
+  Tag,
+  Pagination,
+  Column,
+  VerticalGroup,
+  HorizontalGroup,
+} from '@grafana/ui';
+import { TagBadge } from 'app/core/components/TagFilter/TagBadge';
+import { UserDTO } from 'app/types';
+
+import { Avatar } from './Avatar';
+import { OrgUnits } from './OrgUnits';
+
+type Cell<T extends keyof UserDTO = keyof UserDTO> = CellProps<UserDTO, UserDTO[T]>;
+
+interface UsersTableProps {
+  users: UserDTO[];
+  showPaging?: boolean;
+  totalPages: number;
+  onChangePage: (page: number) => void;
+  currentPage: number;
+}
+
+export const UsersTable = ({ users, showPaging, totalPages, onChangePage, currentPage }: UsersTableProps) => {
+  const showLicensedRole = useMemo(() => users.some((user) => user.licensedRole), [users]);
+  const columns: Array<Column<UserDTO>> = useMemo(
```

Comment:
What do you think about disabling sorting for the table, if it's not actually sorting over multiple pages.... ?

Maybe disable sorting if we know there's multiple pages?

Reply:
> Maybe disable sorting if we know there's multiple pages?

That's a good idea! I'll add this.